pkg/daemon/daemon.go

```diff
@@ -34,6 +36,12 @@ func New(listenAddr string) (srv *Daemon, err error) {
 		return nil, err
 	}
 
+	// Check all runners' health.
```

Comment:
Not sure about that, because we might not have all runners setup for the given environment. When you run testground you generally must have >0 runners working correctly, but not necessarily all of the runners we support.

For example on my local machine I might not have the `cluster:k8s` runner setup.
A remove testground daemon might not have the `local:exec` runner working correctly.

Reply:
I was just following what was decided on #432 for this. I can just remove that part so we can merge for now and decide that later.

Reply:
Is `ListRunners` returning all `testground supported` runners, or all `supposedly configured testground runners in this environment` ?

Reply:
All runners from `EngineConfig` so in this case it's `AllRunners` because we get the default engine.

Reply:
Just removed that bit. Can be done on a separate pr.